scripts/clean-all-caches.sh

```
@@ -0,0 +1,113 @@
+    rm -rf android/.gradle
+    rm -rf android/app/build
+    rm -rf .metro
+    rm -rf ios/build
+    rm -rf ios/Pods
+    rm -rf ios/Podfile.lock
+fi
+
+# Clean sample project caches
+if [[ -d "$PROJECT_ROOT/sample" ]]; then
+    echo "🗂️  Cleaning sample project caches..."
+    cd "$PROJECT_ROOT/sample"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)